charts/jobset/templates/prometheus/service_monitor.yaml

```diff
@@ -30,5 +30,24 @@ spec:
     matchLabels:
       {{- include "jobset.controller.selectorLabels" . | nindent 6 }}
   endpoints:
-  - port: metrics
+  - port: https
+{{- if .Values.certManager.enable }}
```

Comment:
I think, there should be `else` if cert manager is not enabled and it should be `insecureSkipVerify: true`

Reply:
+1, falling back to insecure